extensions/interactions/interaction_specs.json

```diff
@@ -97,6 +97,7 @@
     "default_outcome_heading": null,
     "rule_descriptions": {
       "MatchesExactlyWith": "matches exactly with {{x|NumericExpression}}",
+      "MatchesUpToTrivialManipulations": "matches {{x|NumericExpression}}, taking into account commutativity and associativity.",
```

Comment:
Other rule descriptions don't have a period at the end, so this one shouldn't either. Ditto below.

Reply:
Done.